.github/workflows/test_report.yml

```diff
@@ -30,14 +30,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Download test results to report
-      uses: dawidd6/action-download-artifact@09385b76de790122f4da9c82b17bccf858b9557c # pin@v2
```

Comment:
The specific commit was set here because it might contain insecure codes (it's a personal repository).

Reply:
So we should `pin` to a `specific commit` instead of using something like `v1`, right?

Reply:
Yeah, let's pin the commit version after reviewing if their codes have anything harmful.

Reply:
Okay, let me review its source code very carefully.